docs/schema/resultitemtype-element-arrayofresultitemtype-complextypesps15xsdsearchset2.md

```diff
@@ -15,12 +15,11 @@ description: "Last modified: March 09, 2015"
 
 ## Element information
 
-|||
-|:-----|:-----|
-|**Element type** <br/> |tns:ResultItemType  <br/> |
-|**Namespace** <br/> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration  <br/> |
-|**Schema file** <br/> |schema_Microsoft.Office.Server.Search.Administration.xsd  <br/> |
-
+|Information type|Value|
```

Comment:
Invalid table... please follow same format as other files in this section where te table has column headers & a row to indicate column headers.